src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ToModelTransformers.kt

```diff
@@ -236,6 +237,7 @@ fun toVariation(
   licenceVersion = licence.licenceVersion,
   earliestReleaseDate = earliestReleaseDate,
   isEligibleForEarlyRelease = isEligibleForEarlyRelease,
+  isInHardStopPeriod = false,
```

Comment:
if it doesn't make sense to exist on a variation, don't need to add it to the subclass..

Reply:
Hmm, I thought I tried to just excluded and it complained at me about it being required.

I must be mistaken, I'll remove it now